### PR TITLE
Assignment 5

### PR DESCRIPTION
**Pull request by Sanchitanwar0007**
Screenshot (98)
Assignment 4 completed

### CODE REVIEW

File: src/Assignment-5/assignment5.css [hunk 1]
```diff
@@ -2,6 +2,107 @@
 
 
 .container, .content {
-width: 100%;
-height: 100%;
+height:100%;
+width:100%;
```
Comment (SATISH-SAHANI) [inline src/Assignment-5/assignment5.css:6]: Everything looks perfect but try to write minimal code in css.

**Review by SATISH-SAHANI — APPROVED** [review]
Everything looks perfect but try to write minimal code in css.

File: src/Assignment-5/assignment5.css (hunk 1 above)
Comment (Sanchitanwar0007) [reply src/Assignment-5/assignment5.css:6]: ok

**Comment by gourav81** [comment]
Completed Correctly.

**Pull request merged**